build.sbt

```
@@ -112,6 +113,22 @@ lazy val minitest = crossProject(JVMPlatform, JSPlatform)
 lazy val minitestJVM = minitest.jvm
 lazy val minitestJS = minitest.js
 
+lazy val munit = crossProject(JVMPlatform, JSPlatform)
+  .in(file("munit"))
+  .settings(
+    standardSettings ++ Seq(
+      name := "hedgehog-munit",
+      libraryDependencies += "org.scalameta" %%% "munit" % props.MunitVersion,
```

I think you need to have the following ones to use `@nowarn` in Scala 2.11.
```suggestion
      libraryDependencies ++= Seq("org.scalameta" %%% "munit" % props.MunitVersion) ++
        (if (scalaBinaryVersion.value.startsWith("2.11")) {
          val silencerVersion = "1.7.8"
          Seq(
            "org.scala-lang.modules" %% "scala-collection-compat" % "2.7.0",
            compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
            "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full,
          )
        } else {
          Seq.empty
        }),
```